packages/nimble-components/src/icon-base/index.ts

```diff
@@ -28,8 +28,7 @@ export const registerIcon = (baseName: string, iconClass: IconClass): void => {
     const composedIcon = iconClass.compose({
         baseName,
         template,
-        styles,
-        baseClass: iconClass
```

Comment:
This was for Fast dependency injection (i.e. tagFor) and isn't configured right anyway. The idea is you register a base class to dependency inject the concrete implementation. These `iconClass` are the concrete implementation classes, not base classes, they don't need to be registered to provide for a specific base class. So removed.